### DISCUSSION TITLE
Run code only between certain lines?

### DISCUSSION THREAD

**Discussion by werkstattcodes** [quarto-dev/quarto-cli#7952] [Q&A]
Apologies if this is a very basic question. How can I run the code between two specific lines, e.g. row 10 and row 110 (when they are not in the same code chunk). I am looking for an approach other than manual selecting all rows between the start and the end. As far as I can tell, setting breakpoints is not the right approach. I am using quarto in VS Code. Many thanks.

**Comment by jimjam-slam** [comment]
@werkstattcodes Are you asking how to interactively execute code between lines while you have a Quarto doc open in VSCode, or are you asking how to ensure that only code between two lines is executed when the document renders?

**Reply by werkstattcodes** [reply]
Sorry, should have been more specific. I am asking how to interactively execute code between specific lines (other than manually selecting them). Quite frequently, I want to rerun a part of a script which I am revising and do not want to start the script from the top. Selecting theses lines/chunks is not really helpful.
